R/teal.R

```diff
@@ -14,7 +14,6 @@
 magrittr::`%>%`
 
 # Fix R CMD check notes
-#' @import shiny teal.data teal.transform
-#' @importFrom methods is
```

Comment:
This `import` is not strictly necessary as `methods` is always attached but check will raise a NOTE.